src/python/gem5/prebuilt/demo/x86_demo_board.py

```diff
@@ -77,23 +83,68 @@ def __init__(self):
             "real-world system. Use with caution."
         )
 
-        memory = SingleChannelDDR3_1600(size="2GiB")
+        # The other demo boards have 4 GiB of memory, but X86Board can only
+        # support up to 3 GiB.
+        memory = DualChannelDDR4_2400(size="3GiB")
         processor = SimpleProcessor(
-            cpu_type=CPUTypes.TIMING, isa=ISA.X86, num_cores=4
+            cpu_type=CPUTypes.TIMING, isa=ISA.X86, num_cores=2
         )
-        cache_hierarchy = MESITwoLevelCacheHierarchy(
-            l1d_size="32KiB",
-            l1d_assoc=8,
-            l1i_size="32KiB",
-            l1i_assoc=8,
-            l2_size="1MiB",
-            l2_assoc=16,
-            num_l2_banks=1,
+
+        cache_hierarchy = PrivateL1SharedL2CacheHierarchy(
+            l1d_size="64KiB", l1i_size="64KiB", l2_size="1MiB"
         )
 
         super().__init__(
-            clk_freq="3GHz",
+            clk_freq="1.4GHz",
             processor=processor,
             memory=memory,
             cache_hierarchy=cache_hierarchy,
         )
+
+    # Copied over and adapted from RISCVMatchedBoard and X86Board. We need to
+    # override these functions to get SE mode to work.
```

Comment:
This seems like a good comment to have the commit message, but won't make sense in the context of the code in the future.